themes/health/templates/custom/health-notification.html.twig

```diff
@@ -12,7 +12,7 @@
     <div class="health-notification__icon">
       <div class="alignment-helper">
         {% if icon_url %}
-          <img src={{ icon_url }}>
+          <img src={{ icon_url }} alt="notification-icon">
```

Comment:
@Tjocksmurf - would be more informative to add information on the notification type in the alt text. For example, "Error notification icon".  You can get the notification type from the Notification node.